src/Draco.Compiler/Syntax/Lexer.cs

```diff
@@ -374,6 +374,27 @@ private IToken LexString()
                 }
             }
             offset += mode.ExtendedDelims + 1;
+            // Line continuation
+            var whiteCharOffset = 0;
+            ch = this.Peek(offset);
+            if (char.IsWhiteSpace(ch))
+            {
+                while (char.IsWhiteSpace((ch = this.Peek(offset + whiteCharOffset))) && (ch != '\n')) whiteCharOffset++;
+                if (ch == '\n')
+                {
+                    for (int i = 0; i < whiteCharOffset; i++)
+                    {
+                        this.valueBuilder.Append(' ');
```

Comment:
Do we actually want to append to the value here? Isn't this the "accidental trailing whitespace" that we should ignore?

Reply:
I don't think that we want to ignore it? The user typed it so we add it to the builder.

Reply:
The builder is storing the "significant value" of the tokens, in this case the raw strings that will have to be concatenated. This is why we insert escape sequences already escaped and such